homeassistant/components/sensor/abode.py

```diff
@@ -7,16 +7,18 @@
 import logging
 
 from homeassistant.components.abode import AbodeDevice, DOMAIN as ABODE_DOMAIN
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY, DEVICE_CLASS_LIGHT, DEVICE_CLASS_TEMPERATURE)
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPENDENCIES = ['abode']
 
 # Sensor types: Name, icon
 SENSOR_TYPES = {
-    'temp': ['Temperature', 'thermometer'],
-    'humidity': ['Humidity', 'water-percent'],
-    'lux': ['Lux', 'lightbulb'],
+    'temp': ['Temperature', 'thermometer', DEVICE_CLASS_TEMPERATURE],
+    'humidity': ['Humidity', 'water-percent', DEVICE_CLASS_HUMIDITY],
```

Comment:
can we use `mdi:water-percent` here please. helps us to find all icons easier later

Reply:
I mean we should remove `water-percent` entirely here anyway. The frontend will automatically chose the `mdi:water-percent` icon here when there's a `humidity` device class.

Reply:
I removed the mdi icon here. See the comment from @OttoWinter above. Just haven't pushed the changes yet.